hack/create-nodeport.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+if [ -z $KUBECONFIG ]; then
+  KUBECTL=./cluster/kubectl.sh
+else
+  KUBECTL=kubectl
+fi
+
+NAMESPACE=$($KUBECTL get pod -lk8s-app=secondary-dns -A -o=custom-columns=NS:.metadata.namespace --no-headers)
+POD=$($KUBECTL get pod -lk8s-app=secondary-dns -A -o=custom-columns=NAME:.metadata.name --no-headers)
+
+if [ -z $POD ]; then
+  echo "ERROR: secondary-dns pod not found"
+  exit 1
+fi
+
+${KUBECTL} expose -n ${NAMESPACE} pod ${POD} --name dns-service
+${KUBECTL} expose -n ${NAMESPACE} service/dns-service --name=dns-nodeport --type=NodePort --port 53 --protocol='UDP'
```

Comment:
As far as I know, NodePort can be created directly on top of the pod. Why the intermediate ClusterIP service is needed?

Reply:
If the pod dies, this way is safer from what i understand

Reply:
Changed that it will follow the deployment (because the pod can change)
The service was there so in case the pod is restarted it will still work, but better indeed to follow the deployment
as it is cleaner and serves the same purpose